doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -350,6 +350,35 @@ Matter
 
 The issues in this section are related to the :ref:`ug_matter` protocol.
 
+.. rst-class:: v2-5-0 v2-5-1 v2-5-2 v2-5-3 v2-6-0 v2-6-1
+
+KRKNWK-18916: If the :kconfig:option:`CONFIG_CHIP_LAST_FABRIC_REMOVED_ERASE_AND_PAIRING_START` is selected there are two issues:
+
+  * An assert after removing the last fabric may occurs because OpenThread interface is still active despite of disabling Thread stack.
+  * Cannot commission the device to Matter network because device's MAC address is the same as the saved in SRP server.
```

Comment:
```suggestion
  * The device cannot be commissioned to the Matter network because the device's MAC address is the same as the one saved in the SRP server.
```